wp-content/themes/eighshift-task/src/blocks/custom/image-text/components/image-text-options.js

```diff
@@ -1,12 +1,13 @@
 import { __ } from '@wordpress/i18n';
-import { PanelBody, SelectControl, CheckboxControl } from '@wordpress/components';
+import { PanelBody, SelectControl, ToggleControl } from '@wordpress/components';
 
 export const ImageTextOptions = (props) => {
   const {
-    mediaPosition,
-    onChangeMediaPosition,
-    includeBtn,
-    onChangeIncludeBtn,
+    attributes: {
+      mediaPosition,
+      includeBtn,
+    },
+    actions,
```

Comment:
distracture actions also and use only props values like

onChange={onChangeMediaPosition}